### DISCUSSION TITLE
В RAW-образе станции 3.86 остался выключенным переключатель "эта станция расположена за NAT маршрутизатором"

### DISCUSSION THREAD

**Discussion by atar90** [mikopbx/Core#209] [Ideas (Идеи)]
В версии 3.86. 
В Веб-интерфейсе АТС в разделе "Сетевые интерфейсы" переключатель "**эта станция расположена за NAT маршрутизатором**" сделали по умолчанию включенным. Это удобно. В ISO-образе так.
В RAW-образе станции 3.86 он остался выключенным.

**Comment by boffart** [comment]
У Yandex и Google есть API для получения параметров настройки сети:
http://169.254.169.254/computeMetadata/v1/instance/?recursive=true

Для Azure тоже
http://168.63.129.16/machine

Для mail.ru нет документации на эту тему. Нельзя тупо включить ""эта станция расположена за NAT маршрутизатором" для всех, нужно знать внешний IP.

**Comment by boffart** [comment]
Создал другую задачу
https://github.com/mikopbx/Core/issues/210
